ckanext/datarequests/templates/datarequests/snippets/additional_info.html

```diff
@@ -1,3 +1,4 @@
+{% set dataset_read_route = 'dataset.read' if h.ckan_version() > '2.9' else 'dataset_read' %}
```

Comment:
are you not using your new helper function for this?

Reply:
Good point.